welcome-message/banner.html

```diff
@@ -1,19 +1,21 @@
-<div data-o-component="o-expander" class="o-expander o-expander__content--expanded n-welcome-banner__wrapper" data-o-expander-shrink-to="hidden">
-	<h6 class="n-welcome-banner__heading n-welcome-banner__column n-welcome-banner__column--m">Welcome to the new FT.com.</h6>
-
-	<div class="o-expander__content n-welcome-banner__column n-welcome-banner__column--fill">
+<div class="n-welcome-banner__wrapper">
+	<h6 class="n-welcome-banner__heading n-welcome-banner__column">Welcome to the new FT.com.</h6>
```

Comment:
Guess this is more of a copy question (sorry) - do we definitely want the full stop at the end of this? The 'Ft dot com dot' jars a bit with me